src/main/java/uk/nhs/cdss/config/RESTConfig.java

```diff
@@ -0,0 +1,16 @@
+package uk.nhs.cdss.config;
+
+import org.springframework.boot.web.client.RestTemplateBuilder;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.client.RestTemplate;
+
+@Configuration
+public class RESTConfig {
```

Comment:
Don't think this will be needed in the fhir server

Reply:
Good point, done.